app/controllers/offsprings_controller.rb

```diff
@@ -0,0 +1,56 @@
+class OffspringsController < ApplicationController
+  SCOPE = "activerecord.errors.controllers.offspring".freeze # Necessary to acces locales file
+  def show
+  end
+
+  def index
+  end
+
+  def new
+    @offspring = current_user.offsprings.build
+  end
+
+  def create
+    if access_control?
+      @offspring = current_user.offsprings.build(offsprings_params)
+      unless @offspring.primary_first?
+        flash[:warning] = "Sólo puede añadir a niños de 1º de Primaria"
+        redirect_to static_pages_intructions_path
+        return
+      end
+      save_offspring(@offspring)
+    else
+      flash[:alert] = I18n.t("admin_locked_create", scope: SCOPE)
+    end
+    redirect_to root_path
+  end
+
+  def destroy
+    if access_control?
+      offspring = Offspring.find_by_id(params[:id])
+      offspring.destroy
+      flash[:success] = "Niño borrado"
```

Comment:
Ok thank you for the comments @sergio-ocon . The code was recycled from a previous version of another repo. Im going to edit this file now, since yesterday I tried to build a couple of feature specs.